controllers/tests/controller_server/host_definer/utils/utils.py

```diff
@@ -0,0 +1,206 @@
+import unittest
+from dataclasses import dataclass, field
+import func_timeout
+from munch import Munch
+from mock import patch, Mock
+
+from controllers.servers.host_definer.types import CsiNodeInfo
+import controllers.servers.host_definer.messages as messages
+import controllers.tests.controller_server.host_definer.utils.k8s_manifests_utils as manifest_utils
+import controllers.tests.controller_server.host_definer.settings as settings
+from controllers.servers.host_definer.kubernetes_manager.manager import KubernetesManager
+
+
+@dataclass
+class K8sResourceItems():
+    items: list = field(default_factory=list)
+
+
+class HttpResp():
+    def __init__(self, status, data, reason):
+        self.status = status
+        self.data = data
+        self.reason = reason
+
+    def getheaders(self):
+        return None
+
+
+def get_fake_csi_node_info():
+    csi_node_info = CsiNodeInfo()
```

Comment:
you can mock it with MagicMock with spec

Reply:
removed it